### DISCUSSION TITLE
New XML and XPath parser

### DISCUSSION THREAD

**Discussion by pgundlach** [speedata/publisher#536] [Announcements]
The version 4.15.8 has a major (opt in) change: there is a new XML and XPath parser. The XML parser has been part of the speedata Publisher for some time, though not advertised. The XPath parser will replace the current implementation.

While the existing layouts should work without change, I might have introduced some bugs. Please report them and I will fix them as soon as possible.

The new XML/XPath combination will be the default with version 5 of the speedata Publisher.

Some more information at the [news site](https://news.speedata.de/2023/11/01/newxpathparser/).

**Comment by pr-apes** [comment]
Many thanks for both new parsers, @pgundlach.
